### PR TITLE
Documentation changes

### PR DESCRIPTION
**Pull request by nenad-popovic-91**

### CODE REVIEW

**Comment by nenad-popovic-91** [comment]
I'd say the messages from the last three commits are relevant.

File: plugins/editor/doc/plugin-editor.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+{qmlProperty:int fragmentEnd}
+
+End line of a fragment, in case only a fragment of the document is visible
+
+{qmlProperty:color topColor}
+
+Color of the editor menu
+
+{qmlMethod:hasActiveEditor()}
```
Comment (dinusv) [inline plugins/editor/doc/plugin-editor.md]: `bool hasActiveEditor`

File: plugins/editor/doc/plugin-editor.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+{qmlMethod:forceFocus()}
+
+Force the focus on the editor
+
+{qmlMethod:var cursorWindowCoords()}
+
+Returns the coordinates of the cursor
+
+{qmlType:CodePalette}
+{qmlInherits:external.QtCore#Object}
```
Comment (dinusv) [inline plugins/editor/doc/plugin-editor.md]: `external.QtQml#QtObject`

File: plugins/editor/doc/plugin-editor.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+{qmlProperty:object extension}
+
+Extension for this palette
+
+{qmlSignal:init(var value)}
+
+Signals that the value was initialized
+
+{qmlType:LiveExtension}
+{qmlInherits:external.QtCore#Object}
```
Comment (dinusv) [inline plugins/editor/doc/plugin-editor.md]: `external.QtQml#QtObject`

File: plugins/editor/doc/plugin-editor.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+{qmlProperty:var keyBindings}
+
+Key shortcuts for the given extension
+
+{qmlProperty:var configuration}
+
+Extension configuration
+
+{qmlType:DocumentHandler}
+{qmlInherits:external.QtCore#Object, external.QtQml#ParserStatus}
```
Comment (dinusv) [inline plugins/editor/doc/plugin-editor.md]: ParserStatus is only on the c++ side.

```
{qmlInherits:external.QtQml#QtObject}
```

File: plugins/live/doc/plugin-live.md [hunk 5]
```diff
@@ -11,6 +11,301 @@ import live 1.0
...
+{qmlMethod:readerDataChanged(array data)}
+
+Reacts to data changes inside the file by resyncing local copy of the data.
+
+{qmlSignal:init()}
+
+Called when the loader is initialized (component complete).
+
+{qmlType:VisualLogFilter}
+{qmlInherits:VisualLogBaseModel,external.QtQml#ParserStatus}
```
Comment (dinusv) [inline plugins/live/doc/plugin-live.md]:
```
{qmlInherits:base#VisualLogBaseModel}
```

File: plugins/live/doc/plugin-live.md [hunk 6]
```diff
@@ -11,6 +11,301 @@ import live 1.0
...
+{qmlProperty:string source}
+
+The source code of the item
+
+{qmlProperty:object item}
+
+The created item
+
+{qmlType:ComponentSource}
+{qmlInherits: QtCore#Object,QtQml#ParserStatus}
```
Comment (dinusv) [inline plugins/live/doc/plugin-live.md]:
```
{qmlInherits:external.QtQml#QtObject}
```

File: plugins/live/doc/plugin-live.md [hunk 7]
```diff
@@ -11,6 +11,301 @@ import live 1.0
...
+{qmlProperty:color backgroundColor}
+
+Color of the button
+
+{qmlSignal: clicked()}
+
+Signal that we've clicked the button
+
+{qmlType:VisualLogView}
+{qmlInherits:external.QtQuick#ScrollView}
```
Comment (dinusv) [inline plugins/live/doc/plugin-live.md]:
```
{qmlInherits:external.QtQuick.Controls#ScrollView} 
```

**Pull request merged**